WcaOnRails/config/locales/en.yml

```diff
@@ -134,6 +134,8 @@ en:
         contact: "Contact"
         information: "Informations"
 
+        base_entry_fee_in_cents: "Base entry fee in cents"
```

Comment:
As you said, the word "cents" doesn't make sense for "Yen". How about we call it "currency unit", and have a "currency_unit_description" in our database? That way when Yen is chosen, we could pass in the unit here, and render "Base entry fee in Yen" to the user, which seems a lot nicer than "Base entry fee in cents".